09-features.md

```diff
@@ -40,8 +40,7 @@ The Context column decodes as follows:
 | 18/19 | `option_support_large_channel`   | Can create large channels                                 | IN       |                   | [BOLT #2](02-peer-protocol.md#the-open_channel-message) |
 | 20/21 | `option_anchor_outputs`          | Anchor outputs                                            | IN       | `option_static_remotekey` | [BOLT #3](03-transactions.md)         |
 | 22/23 | `option_anchors_zero_fee_htlc_tx` | Anchor commitment type with zero fee HTLC transactions   | IN       |                   | [BOLT #3][bolt03-htlc-tx], [lightning-dev][ml-sighash-single-harmful]|
-| 26/27 | `option_shutdown_anysegwit`         | Future segwit versions allowed in `shutdown`              | IN       |                   | [BOLT #2][bolt02-shutdown]   |
-| 28/29 | `option_dual_fund`             | Use v2 of channel open, enables dual funding              | IN      | `option_anchor_outputs`, `option_static_remotekey`   | [BOLT #2](02-peer-protocol.md)        |
+| 28/29 | `option_dual_fund`             | Use v2 of channel open, enables dual funding              | IN      | `option_anchors_zero_fee_htlc_tx`   | [BOLT #2](02-peer-protocol.md)        |
```

Comment:
h/t to @ddustin for assistance w/ review <3

Reply:
I don't know why I removed `option_shutdown_anysegwit` there, restored in https://github.com/niftynei/lightning-rfc/pull/3/commits/37491624085d967f9a36372d9112d1d3589607e5